introduction.md

```diff
@@ -0,0 +1 @@
+Those who are possessed by nothing possess everything. -Morihei Ueshiba
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.